tests/test_openapi_async.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+
+
+def test_allow_async_openapi():
+    async def async_openapi():
+        return {"foo": "bar"}
+
+    mod_app = FastAPI()  # use fresh instance to not affect other tests
```

Comment:
```suggestion
    mod_app = FastAPI()
```

This comment is not needed anymore